ts/WoltLabSuite/Core/Ui/Poll/Manager/View/Abstract.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Abstract implementation for poll views.
+ *
+ * @author  Joshua Ruesweg
+ * @copyright  2001-2022 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Ui/Poll/Manager/View/Abstract
+ * @since   5.5
+ */
+
+import * as Core from "../../../../Core";
+import AjaxRequest from "../../../../Ajax/Request";
+import { RequestOptions, ResponseData } from "../../../../Ajax/Data";
+import Manager, { PollViews } from "../Manager";
+
+export abstract class Abstract {
```

Comment:
I don't think that this module is required. If you just need 2 or 3 helper functions, just use a "util" style module with exports and go for composition for the views.